CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-# 4.7.1
+# 4.7.1 (Unreleased)
```

Comment:
Lets mark this release as 5.0 as it will be a major change because of design changes